packages/api-kintone/src/invoiceB2B/getAllInvoiceB2B.test.ts

```diff
@@ -0,0 +1,18 @@
+import { describe, expect, it } from '@jest/globals';
+import { getAllInvoiceB2B } from './getAllInvoiceB2B';
+
+describe('getAllInvoiceB2B', () => {
+  it('should return all invoiceB2B', async () => {
+    const result = await getAllInvoiceB2B();
+
+    console.log('result: ', result.length);
+
+    // each record should have $id field
+    expect(result).toEqual(
+      expect.arrayContaining([
+        expect.objectContaining({ $id: expect.any(String) }),
```

Comment:
```suggestion
        expect.objectContaining({
          $id: {
            type: expect.any(String),
            value: expect.any(String),
          },
        }),
```
テストがエラーになってしまいました。
修正案です。

Reply:
ありがとうございます！

コミットが一個別にPRにやってしまいました。すみませんが、ご確認お願いします。

GithubのPRのスタックが混乱中です。ｗ

Reply:
修正ありがとうございます！
急ぎの対応が多いですもんねf^_^;
ご無理なさらず！

Reply:
予定した分だけ、趣味的にやっています。ｗ

プロジェクト管理も無理はスケジュールにならないようにしましょう。
早川さんも調整してもいいですよ。(普段PMがやるので、個人的に管理時間や雑談も含めて調整していますｗ)

Estimateフィールドもありますが、開発者全員の感覚で調整するのは一般的です。

Reply:
わかりました！
今プロジェクト管理はほぼお任せしてしまっているので、
自分の担当分は分かりやすいので、まずはそちらから調整してみますね！